rfcdiff

```diff
@@ -101,10 +101,10 @@
 #		rfcdiff first-file second-file
 #	
 #	The latest version is available from
-#		http://tools.ietf.org/tools/rfcdiff/
+#		https://author-tools.ietf.org/rfcdiff
```

Comment:
The tools,ietf.org link pointed to a meta-discussion page - more like what a README.md at a github repo should do (not that the rfcdiff project's readme does this right now).

I think this should point to the repo. We could, separately, update the README.md there, but lets not spend a lot of effort there.

Reply:
For reference: https://web.archive.org/web/20190904191646/https://tools.ietf.org/tools/rfcdiff/

Reply:
@rjsparks https://github.com/ietf-tools/rfcdiff/pull/7/commits/ddd7f706d2f218c00eee31d1700518cd407d5736 changes the URLs to this repository.